index.ts

```diff
@@ -158,19 +152,23 @@ export const Record = {
 export async function endpoint({ args: { path, query, headers, body } }) {
   switch (path) {
     case "/incoming-webhook": {
-      let event = JSON.parse(body);
-      const config = state.webhooks.find((w) => w.id === event.webhook.id);
+      const event = JSON.parse(body);
+      const webhookId = event.webhook.id;
+      const config = state.webhooks[webhookId];
+      if (!config) {
+        console.log(`Webhook ${webhookId} is not part of this program`);
+        return;
```

Comment:
You should `throw` instead of `console.log` here. That way Airtable gets a 500 and eventually stops sending that webhook

Reply:
I had placed it that way.